### ISSUE TITLE
[enhancement]: Add support for C220 Cameras

### ISSUE THREAD

**Issue by MizterOz** [homebridge-eufy-security/plugin#647]
### Describe the solution you'd like

Please can you add support for the C220 pan and tilt cameras
![IMG_0330](https://github.com/user-attachments/assets/228e4884-b8ae-4359-9721-8fb1605dc2d5)

**Comment by MizterOz** [comment]
Dead link

**Comment by arktiknova** [comment]
Yeah deadl link. Btw this model is kinda new too that is not in the list, the Indoor Camera C220 2K pan and tilt.

https://www.eufy.com/products/t8w11121?ref=navimenu_2_2_3_img&fliter_category=indoor-security-camera

**Comment by MizterOz** [comment]
Thank you for getting back. Is this something that will be compatible soon do you know? Thank you.

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
